### PR TITLE
adding icon to header and pop up to new event

### PR DESCRIPTION
**Pull request by okediran**
1. I adding icon to header
2. Highlight days with existing Event in the small calendar

### CODE REVIEW

File: src/components/calendar/AppointmentFormContainerBasic.jsx [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+              </Button>
+              <div>
+                <Button
+                  aria-controls="outlook-dropdown"
+                  aria-haspopup="true"
+                  // onClick={handleClick}
+                >
+                  Open Outlook Dropdown
+                </Button>
+                <Menu
```
[the remark this answers is not in the document]
Comment (okediran) [reply src/components/calendar/AppointmentFormContainerBasic.jsx:278]: it meant for dropdown

File: src/components/calendar/AppointmentFormContainerBasic.jsx [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+                    endAdornment: (
+                      <InputAdornment position="end">
+                        <AntSwitch
+                          inputProps={{ "aria-label": "ant design" }}
+                        />
+                        <i
+                          data-icon-name="IcFluentOfficeSkypeColor"
+                          aria-hidden="true"
+                        >
+                          <svg
```
[the remark this answers is not in the document]
Comment (okediran) [reply src/components/calendar/AppointmentFormContainerBasic.jsx:354]: i dont understand the first paragraph sir

**Comment by StefanAkie** [comment]
Why did you close threads without actually resolving them

File: src/components/calendar/AppointmentFormContainerBasic.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (StefanAkie) [reply src/components/calendar/AppointmentFormContainerBasic.jsx]: <svg> tags should not be randomly dumped into container files. please move them into another file

**Comment by StefanAkie** [comment]
Nice job, could you re-add the TinyMCE translation files, so they are not deleted? You can just check them out from the master branch and push the files onto this branch.

**Comment by okediran** [comment]
ok i will

**Pull request merged**